bacom-blog/article-feed/article-feed.js

```diff
@@ -0,0 +1,77 @@
+import { selectAllBlocks, moveNode } from '../../utils/mdast-utils.js';
+import { STATUS_SKIPPED, STATUS_SUCCESS } from '../../utils/migration-utils.js';
+
+function getFeedRow(feedURL) {
+    return {
+        type: 'gtRow',
+        children: [
+            {
+                type: 'gtCell',
+                children: [
+                    {
+                        type: 'paragraph',
+                        children: [
+                            {
+                                type: 'text',
+                                value: 'Feed'
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                type: 'gtCell',
+                children: [
+                    {
+                        type: 'paragraph',
+                        children: [
+                            {
+                                type: 'text',
+                                value: feedURL
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    };
+}
+
+/**
+ * Update AU and UK Article Feeds
+ * 
+ * @param {object} mdast - markdown tree
+ * @param {string} entry - entry path
+ * @returns {object} - report
+ */
+export default function updateArticleFeed(mdast, entry) {
+    const AU_PATH = '/au/blog/';
+    const UK_PATH = '/uk/blog/';
+    let feedURL;
+
+    console.log(entry);
+
+    if (entry.includes(AU_PATH)) feedURL = 'https://main--bacom-blog--adobecom.hlx.page/au/us-au-query-index.json';
+    if (entry.includes(UK_PATH)) feedURL = 'https://main--bacom-blog--adobecom.hlx.page/uk/us-uk-query-index.json';
+    if (!feedURL) {
```

Comment:
nit: using both inline and nested if statements, good case for using if...else statements.
```suggestion
    if (entry.includes(AU_PATH)) {
      feedURL = 'https://main--bacom-blog--adobecom.hlx.page/au/us-au-query-index.json';
    } else if (entry.includes(UK_PATH)) {
      feedURL = 'https://main--bacom-blog--adobecom.hlx.page/uk/us-uk-query-index.json';
    } else {
```

Reply:
It's not nested, so not sure what you mean there. I think it's fine as is so I'm going to leave it